static/app/components/codecov/integratedOrgPicker/integratedOrgPicker.tsx

```diff
@@ -0,0 +1,36 @@
+import {useSearchParams} from 'react-router-dom';
+
+import {useCodecovContext} from 'sentry/components/codecov/context/codecovContext';
+import {t} from 'sentry/locale';
+
+import {
+  IntegratedOrgSelector,
+  type IntegratedOrgSelectorProps,
+} from './integratedOrgSelector';
+
+export interface IntegratedOrgProps extends Partial<IntegratedOrgSelectorProps> {}
+
+export function IntegratedOrgPicker({
+  triggerProps = {},
+  ...selectProps
+}: IntegratedOrgProps) {
+  const {integratedOrg} = useCodecovContext();
```

Comment:
I actually just realized we probably can use this value inside the Selector component and not the context, so we should be able to move it down there and prevent an unnecessary render